ds_judgements_public_ui/templates/pages/open_justice_licence.html

```diff
@@ -4,165 +4,176 @@
         content="Open justice is a fundamental constitutional principle and necessary for the rule of law. The purpose of this licence is to support open justice." />
 {% endblock meta_description %}
 {% block static_content %}
-  <h1>Open Justice Licence</h1>
-  <div class="anchor-links">
-    <h3 class="anchor-links__header">Information on this page:</h3>
-    <ul class="anchor-links__list">
-      <li class="anchor-links__list-option">
-        <a href="#section-purpose">Purpose of this licence</a>
-      </li>
-      <li class="anchor-links__list-option">
-        <a href="#section-information">Using Information under this licence</a>
-      </li>
-      <li class="anchor-links__list-option">
-        <a href="#section-exclusions">Exclusions</a>
-      </li>
-      <li class="anchor-links__list-option">
-        <a href="#section-no-warranty">No warranty</a>
-      </li>
-      <li class="anchor-links__list-option">
-        <a href="#section-governing">Governing Law</a>
-      </li>
-      <li class="anchor-links__list-option">
-        <a href="#section-definitions">Definitions</a>
-      </li>
-      <li class="anchor-links__list-option">
-        <a href="#section-about">About the Open Justice Licence</a>
-      </li>
-    </ul>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <div class="govuk-grid-column-one-third">
+        <h1 class="hidden-h1-mobile">Open Justice Licence</h1>
+        <div class="anchor-links">
+          <h3 class="anchor-links__header">On this page</h3>
+          <ul class="anchor-links__list">
+            <li class="anchor-links__list-option">
+              <a href="#section-purpose">Purpose of this licence</a>
+            </li>
+            <li class="anchor-links__list-option">
+              <a href="#section-information">Using Information under this licence</a>
+            </li>
+            <li class="anchor-links__list-option">
+              <a href="#section-exclusions">Exclusions</a>
+            </li>
+            <li class="anchor-links__list-option">
+              <a href="#section-no-warranty">No warranty</a>
+            </li>
+            <li class="anchor-links__list-option">
+              <a href="#section-governing">Governing Law</a>
+            </li>
+            <li class="anchor-links__list-option">
+              <a href="#section-definitions">Definitions</a>
+            </li>
+            <li class="anchor-links__list-option">
+              <a href="#section-about">About the Open Justice Licence</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div class="govuk-grid-column-two-thirds">
+        <h1 class="hidden-h1-desktop">Open Justice Licence</h1>
```

Comment:
I wonder if the text of the licence itself could be made into an include, because the exact same text is used here and in `start.html`?